cmd/cluster/context_test.go

```diff
@@ -592,3 +593,41 @@ func TestPrintUserBannedStatus(t *testing.T) {
 		})
 	}
 }
+func TestPrintMigrationStatus(t *testing.T) {
+	tests := []struct {
+		name           string
+		data           contextData
+		expectedOutput string
+	}{
+		{
```

Comment:
I don't think you _have_ to do this here if you don't want to, but you could mock the OCM client API responses and then test that given certain responses, you print certain output. That would also help you validate that whole code flow without having an SDN to OVN migration currently in flight in prod/stage.